tox.ini

```diff
@@ -2,10 +2,16 @@
 envlist = cov-init, py27, py33, py34, py35, py36, docs, packaging, cov-report
 
 [testenv]
+passenv = CI TRAVIS TRAVIS_*
+whitelist_externals =
+    bash
+    make
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/cli_helpers
     COVERAGE_FILE = .coverage.{envname}
-commands = ./setup.py test --coverage lint
+commands =
+    ./setup.py test --coverage lint
+    bash -c 'if [ -n "$TRAVIS" ]; then {envbindir}/coverage xml && {envbindir}/codecov; fi'
```

Comment:
Hmm interesting, does this automatically upload the coverage stats to codecov? Maybe `TRAVIS` should be changed to `CODECOV`?